astropy/coordinates/solar_system.py

```diff
@@ -80,7 +81,10 @@ class solar_system_ephemeris(ScienceState):
     This can be one of the following::
```

Comment:
Maybe try this, build the doc locally and see if those warnings go away?
```suggestion
    This can be one of the following:
```